src/helpers/viewPointHelper.ts

```diff
@@ -0,0 +1,38 @@
+import {
```

Comment:
I guess the name should be `viewpointHelper.ts` to match with the VCS map core naming.